src/plugins/workspace/server/plugin.ts

```diff
@@ -108,6 +109,26 @@ export class WorkspacePlugin implements Plugin<WorkspacePluginSetup, WorkspacePl
     });
   }
 
+  private setUpRedirectPage(core: CoreSetup) {
+    core.http.registerOnPostAuth(async (request, response, toolkit) => {
+      const path = request.url.pathname;
+      if (path === '/') {
+        const listResponse = await this.client?.list(
+          { request, logger: this.logger },
+          { page: 1, perPage: 1 }
+        );
+        if (listResponse?.success && listResponse.result.total > 0) {
+          return toolkit.next();
+        }
+        const basePath = request.url.origin;
```

Comment:
Shall we use `coreSetup.http.basePath.serverBasePath` instead of `request.url.origin`. I think it won't work fine without `--no-base-path` when `yarn start`.

Reply:
Sure, I will update the basepath